src/main/java/com/chopshop166/chopshoplib/maps/DifferentialDriveMap.java

```diff
@@ -0,0 +1,22 @@
+package com.chopshop166.chopshoplib.maps;
+
+import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
+
+import edu.wpi.first.wpilibj.GyroBase;
+
+/**
+ * Differential Drive Map
+ * <p>
+ * Gets the Left {@link SendableSpeedController}, gets the right
```

Comment:
Maybe say that it "provides the ..."

Reply:
Ideally describe what it does at a high level - provide a base map for a differential drive chassis. Then provide function documentation to clarify what each one does, instead of a paragraph below.